### PR TITLE
HW 3

### PR DESCRIPTION
**Pull request by mpb2882**

### CODE REVIEW

File: 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+rds.t <- arrange(rds, Observer)
+dev1.means  <- data.frame()
+dev2.means  <- data.frame()
+dev1.sd     <- data.frame()
+dev2.sd     <- data.frame()
+dev12.cor   <- data.frame()
+
+#Set a repeating loop for 1:13
+for(i in 1:13){
+    #Calculate the mean for each Observer using i
```
Comment (rsettlage) [inline 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: or make this a function that gets called in the loop...

File: 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Patient <- as_tibble(rep(c(1:15), times=6))
+measurementTime <- as_tibble(rep(c("Morning 1", "Afternoon 1", "Evening 1", "Morning 2", "Afternoon 2", "Evening 2"), each = 15))
+
+bloodPressureFinal <- cbind(Patient, measurementTime, bloodPressureTidy)
+colnames(bloodPressureFinal) <- c("Patient", "Measurement Time", "Blood Pressure")
+
+
+invisible(as_tibble(bloodPressureFinal))
+invisible(bloodPressureByPatient <- arrange(bloodPressureFinal, Patient))
```
Comment (rsettlage) [inline 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: cool, a new function for me, `invisible`

The way I do this is by dividing up my code a bit more to allow annotation.  All code I want to just run gets the echo=F, all code I would like to make it into my document goes in a echo=F, include=T code chunk.

File: 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    print("Gebreken")
+    print(Car_Gebreken_raw.colclass)
+
+    print("Geconstat")
+    print(Car_Geconstat_raw.colclass)
+
+    print("Personen")
+    print(Car_Person_raw.colclass)
+    
+#Beyond loading the data, I'm really not sure what to do for this question.
```
Comment (rsettlage) [inline 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: You should come see me.  This is the important one...

Comment (rsettlage) [reply 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: When you come see me, make sure we look at the pdf too...I may have some comments about that as well.

File: 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd (hunk 2 above)
Comment (mpb2882) [reply 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: Thanks for the tip on how you do it!

File: 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd (hunk 3 above)
Comment (mpb2882) [reply 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: I can definitely do that.  I'm going to take a look at it tomorrow morning with a blank slate on it, and I will let you know if I can't figure out something to do with this one.

Comment (rsettlage) [reply 03_good_programming_R_functions/HW3_Blevins_Matthew.Rmd]: Cool.  Put on a piece of paper what information you have in the 3 datasets and then decide what you need to get from which dataset and when.  There are only about 5 steps.

**Pull request closed without merging**